### ISSUE TITLE
Missing dependency: babel-core

### ISSUE THREAD

**Issue by vdel26** [netlify/victor-hugo#55]
I just did a fresh install of the project and got the following error:

```
ERROR in ./src/js/app.js
Module build failed: Error: Cannot find module 'babel-core'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/victordg/Code/escorpora/node_modules/babel-loader/lib/index.js:3:13)
    at Module._compile (module.js:635:30)
    at Module._extensions..js (module.js:646:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/victordg/Code/escorpora/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
 @ multi ./js/app
```
It was fixed after running `npm install babel-core --save`. 
Looks like it should be added to the list of dependencies.

----

INFO:
node version: v8.9.1
npm version: 5.5.1

**Comment by erquhart** [comment]
Fixed in #51.

**Comment by Rajesh2github** [comment]
thanks,
